app/assets/javascripts/components/QuillEditor.js

```diff
@@ -17,17 +17,17 @@ export default class QuillEditor extends React.Component {
       modules: {
         toolbar: [
           ['bold', 'italic', 'underline'], //, 'strike'
-          // ['blockquote', 'code-block'],
-          // [{ 'header': 1 }, { 'header': 2 }],
           [{ 'list': 'ordered'}, { 'list': 'bullet' }],
           [{ 'script': 'sub'}, { 'script': 'super' }],
-          [{ 'indent': '-1'}, { 'indent': '+1' }],
-          // [{ 'direction': 'rtl' }],
-          // [{ 'size': ['small', false, 'large', 'huge'] }],
           [{ 'header': [1, 2, 3, 4, 5, 6, false] }],
           [{ 'color': [] }, { 'background': [] }],
-          [{ 'font': [] }],
-          [{ 'align': [] }],
+          // [{ 'font': [] }],
```

Comment:
We can uncomment a single line to add new format to the editor so that I don't use block comment